### ISSUE TITLE
Goroma Boat malformed map?

### ISSUE THREAD

**Issue by Sixish** [tibiamaps/tibia-map-data#41]
https://tibiamaps.io/map#32162,32574,6:2

Looks like two maps were incorrectly merged.

**Comment by mathiasbynens** [comment]
FWIW, the actual merge happened correctly — it’s just that the Odyssey data contains lots of glitched map parts like that. (I’ve been wondering what caused that — malformed cams, or a bug in a script somewhere?) https://raw.githubusercontent.com/Sixish/Odyssey/master/Odyssey/Images/Minimap/Map06.png has the same bug.

Fixing…

**Comment by Sixish** [comment]
The errors in Odyssey's map data were caused by the variation in map packets across client versions. Unfortunately, a lot of the map is "niche" areas that don't get visited anymore. I had to go back as early as 7.4 to get data from the map. TibiaCast recordings have a (TibiaCast) version header which can be used to approximate the client version, but in some cases the map packets were structured differently between two client versions of the same TibiaCast version.
